Lib/annotationlib.py

```diff
@@ -0,0 +1,635 @@
+"""Helpers for introspecting and wrapping annotations."""
+
+import ast
+import enum
+import functools
+import sys
+import types
+
+
```

Comment:
Can you perhaps add an `__all__` export? Because otherwise, I would think that `ForwardRef` is public but your comment tells me it's an "internal" wrapper, so it could be confusing.

Reply:
I will add `__all__`. `ForwardRef` is public, I'll adjust the docstring.